Sources/NIOKit/Extensions/EventLoop/EventLoop+WhenAll.swift

```diff
@@ -0,0 +1,65 @@
+import NIO
+
+extension EventLoop {
```

Comment:
The primary difference is that these methods returns a `Result<T, Error>` for each of the ELFs, rather than acting as just a notification of when they completed.